code/game/machinery/washing_machine.dm

```diff
@@ -234,6 +234,9 @@ GLOBAL_LIST_INIT(dye_registry, list(
 	WM.bloody_mess = TRUE
 	gib()
 
+/obj/item/organ/brain/machine_wash(obj/machinery/washing_machine/WM)
+	brainwash(brainmob, pick(world.file2list("strings/abductee_objectives.txt")))
```

Comment:
might we need to have a maximum brainwashing? It will allow to have brainwash objectives again and again as long as you wash theirs.

Reply:
Good point. I'll fix it tomorrow, if I remember.

Reply:
hmm, we might need to consider a case they have a traitor brainwash.. wipe brainwash then re-brainwash with abductor's?

Reply:
I guess logically it should change the objective to abductee. Also if the brain is being rewashed, it should become smooth with respective traumas.